smartcontract/service/native/governance/method.go

```diff
@@ -913,13 +915,16 @@ func executeSplit2(native *native.NativeService, contract common.Address, view u
 
 	//fee split of consensus peer
 	for i := 0; i < int(config.K); i++ {
-		nodeAmount := nodeIncome * uint64(globalParam.A) / 100 * peersCandidate[i].S / sumS
-		err = splitNodeFee(native, contract, peersCandidate[i].PeerPubkey, peersCandidate[i].Address, true,
-			peerPoolMap.PeerPoolMap[peersCandidate[i].PeerPubkey].TotalPos, nodeAmount)
+		nodeAmount := new(big.Int).Div(new(big.Int).Mul(new(big.Int).Div(new(big.Int).Mul(nodeIncome,
+			new(big.Int).SetUint64(uint64(globalParam.A))), new(big.Int).SetUint64(100)),
```

Comment:
can this be simplified? it is hard to read.

Reply:
yes. 
Or left the old code as comment, to help reading.